packages/mwp-logger-plugin/src/logger.js

```diff
@@ -1,12 +1,55 @@
 import bunyan from 'bunyan';
+import bunyanDebugStream from 'bunyan-debug-stream';
 import LoggingBunyan from '@google-cloud/logging-bunyan';
 
-const streams = [{ stream: process.stdout }]; // stdout by default
-const { GAE_INSTANCE, GCLOUD_PROJECT, GAE_VERSION, GAE_SERVICE } = process.env;
+const formatDuration = ms => {
+	const seconds = Math.floor(ms / 1000);
+	const nanos = ms % 1000 * 1000 * 1000;
```

Comment:
I don't really understand what this is for, but I assume we copied it from somewhere and it is what we want?

Reply:
yeah, I'll add comments. It's converting the millisecond response time that we get from Hapi into a `{ seconds, nanos }` object that is required by the 'latency' logger in [Stackdriver's HttpRequest](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#httprequest) param

Specifically, it is a [Duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration), which seems like an amazingly awkward definition of time span.